initializers/db.go

```diff
@@ -0,0 +1,42 @@
+package initializers
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"tiny-site-backend/models"
+
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+)
+
+var DB *gorm.DB
+
+func ConnectDB(config *Config) {
+	var err error
+	dsn := fmt.Sprintf("host=%s user=%s password=%s dbname=%s port=%s sslmode=disable TimeZone=Asia/Shanghai", config.DBHost, config.DBUserName, config.DBUserPassword, config.DBName, config.DBPort)
+
+	DB, err = gorm.Open(postgres.Open(dsn), &gorm.Config{})
+	if err != nil {
+		log.Fatal("Failed to connect to the Database! \n", err.Error())
+		os.Exit(1)
```

Comment:
Hi @vinit717 using `os.Exit(1)` will terminate the entire process, this can also affect the other clients running. So I suggest throwing error to the user rather than calling `os.Exit(1)`

Reply:
removed